modules/cnf-associate-secondary-interfaces-metrics-to-network-attachments.adoc

```diff
@@ -0,0 +1,86 @@
+// CNF-43 Associate Secondary Interfaces Metrics to Network Attachments
```

Comment:
This is "generic openshift", not related only to CNF
I think the best place is under the multiple network section, which is what these metrics are about https://docs.openshift.com/container-platform/4.5/networking/multiple_networks